cleanup.js

```diff
@@ -0,0 +1,53 @@
+const SQL = require('@nearform/sql')
+const { getDatabase, getExpiryConfig, runIfDev } = require('./utils')
+
+async function createRegistrationMetrics(client) {
+  const sql = SQL`
+    INSERT INTO metrics (date, event, os, version, value)
+    SELECT CURRENT_DATE, 'REGISTER', '', '', COUNT(id)
+    FROM registrations WHERE created_at::DATE = CURRENT_DATE
+    ON CONFLICT ON CONSTRAINT metrics_pkey
+    DO UPDATE SET value = EXCLUDED.value
+    RETURNING value
+  `
+
+  const { rows } = await client.query(sql)
+  const [{ value }] = rows
+
+  console.log(`updated register metric for today with value ${value}`)
+}
+
+async function removeExpiredCodes(client, codeLifetime) {
+  const sql = SQL`
+    DELETE FROM verifications
+    WHERE created_at < CURRENT_TIMESTAMP - ${`${codeLifetime} mins`}::INTERVAL
```

Comment:
This needs to be a longer interval as we want users to get the expired code messages.

Reply:
That's fair enough - shall we hard code or create another param in infra to manage this? Or we could just keep the code around for `codeLifetime * 3` or something similar.

Reply:
I think a separate param to configure how long we keep for.